domain/model/pokemon.go

```diff
@@ -0,0 +1,44 @@
+package model
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
```

Comment:
Why did you decide to include this constant here? 
Just wondering.

Reply:
It was a remnant from previous design I was working, but I moved to the config struct since I don't have anything relate to a store I thought that maybe this was the best place to store this info.